chanel_1:
  frequency: 15  # Setting to high frequency caused lags in the simulation
  world_name: husarion_world
  parent_link: panther
  position: [0.362, 0.0, 0.201]
  orientation: [0.0, 0.0, 0.0]
  led_strip_width: 0.5
  topic: lights/channel_1_frame
  light_name: front_light
  number_of_leds: 46

chanel_2:
  frequency: 15  # Setting to high frequency caused lags in the simulation
  world_name: husarion_world
  parent_link: panther
  position: [-0.362, 0.0, 0.201]
  orientation: [0.0, 0.0, 0.0]
  led_strip_width: 0.5
  topic: lights/channel_2_frame
  light_name: rear_light
  number_of_leds: 46
